.editorconfig

```diff
@@ -10,5 +10,5 @@ max_length = 120
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.{json, yml, html, mustache, css}]
+[*.{json,yml,html,mustache,css}]
 indent_size = 2
```

Comment:
@kunagpal Do you mean this should actually be `4`?

Reply:
My bad, I misread your previous comment. The indentation for inbuilt JSON files is 2, but 4 for generated JSON files (like the one above)